14 create database init file

Marcus-Aastum opened the pull request:
Create initialization files so that tables are set up on startup in the database

#14

Marcus-Aastum committed:
Create SQL init file

Create SQL file that creates tables with collumns

Create database init file #14

Marcus-Aastum committed:
Fix SQL bug

Fix SQL bug where database did not exist when creating tables

Create database init file #14

Sigdriv commented on Database/mssql/scripts/init_tables.sql:
```diff
@@ -0,0 +1,19 @@
+    ID INT IDENTITY(1,1) PRIMARY KEY,
+    title VARCHAR(100),
+    body TEXT,
+    startdate DATE,
+    starttime TIME,
+    enddate DATE,
+    endtime TIME,
```
Æ d nødvendig å ha `startdate` og `starttime`? Æ d it nok mæ bare en `startdate`som inneheld startdate og starttime i d hær formatet `2024-04-29T11:00:00Z`?

Sigdriv reviewed (approved):
Sjekke opp den ene kommentaren, men ellers ser det bra ut

Marcus-Aastum committed:
Change datatype

Change datatype to remove unnecessary collumns in Events-table

Create database init file #14
